### PR TITLE
Logout functionality

### PR DESCRIPTION
**Pull request by KiranVarma02**
### Change description ###
Logout functionality

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ *] No
```

### CODE REVIEW

File: src/main/routes/logout.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      if(jwt) {
+        try {
+          IdamClient.invalidateSession(jwt);
+        } catch (err) {
+          logger.error(`Failed invalidating JWT for user  ${user}`);
+        }
+      }
+
+      res.clearCookie(sessionCookie);
+      res.redirect(Paths.homePage.uri);
```
Comment (Jamesnelson117) [inline src/main/routes/logout.ts]: Why not just redirect to landing?

File: src/main/routes/logout.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as express from 'express';
+
+import { Paths } from 'main/app/paths';
+import { JwtExtractor } from 'idam/jwtExtractor';
+import { User } from 'idam/user';
+import { IdamClient } from 'idam/idamClient';
+import { Logger } from '@hmcts/nodejs-logging';
+
+import config from 'config';
```
Comment (Jamesnelson117) [inline src/main/routes/logout.ts:9]: Minor but what do you think about keeping node modules above modules we author?

File: src/main/modules/nunjucks/index.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import * as path from 'path';
 import * as express from 'express';
 import * as nunjucks from 'nunjucks';
 import { i18n } from 'i18next';
+import { Paths as AppPaths } from 'main/app/paths';
```
Comment (Jamesnelson117) [inline src/main/modules/nunjucks/index.ts:5]: Why the re-assign?

File: src/main/routes/logout.ts (hunk 1 above)
Comment (KiranVarma02) [reply src/main/routes/logout.ts]: that's fine too. Done

File: src/main/routes/logout.ts (hunk 2 above)
Comment (KiranVarma02) [reply src/main/routes/logout.ts:9]: I changed the order for now. Personally i only care about grouping on import specific, import * or import {} as i have done and leave the rest for intellij auto import.

File: src/main/modules/nunjucks/index.ts (hunk 3 above)
Comment (KiranVarma02) [reply src/main/modules/nunjucks/index.ts:5]: we will import case paths too in future and i want to import that as CasePaths.

**Review by Jamesnelson117 — APPROVED** [review]
LGTM

**Review by mounikahmcts — APPROVED** [review]

**Pull request merged**